lib/Cake/Network/CakeSocket.php

```diff
@@ -93,10 +94,14 @@ class CakeSocket {
 		'sslv3_client' => STREAM_CRYPTO_METHOD_SSLv3_CLIENT,
 		'sslv23_client' => STREAM_CRYPTO_METHOD_SSLv23_CLIENT,
 		'tls_client' => STREAM_CRYPTO_METHOD_TLS_CLIENT,
+		'tlsv1_1_client' => STREAM_CRYPTO_METHOD_TLSv1_1_CLIENT,
+		'tlsv1_2_client' => STREAM_CRYPTO_METHOD_TLSv1_2_CLIENT,
 		'sslv2_server' => STREAM_CRYPTO_METHOD_SSLv2_SERVER,
 		'sslv3_server' => STREAM_CRYPTO_METHOD_SSLv3_SERVER,
 		'sslv23_server' => STREAM_CRYPTO_METHOD_SSLv23_SERVER,
-		'tls_server' => STREAM_CRYPTO_METHOD_TLS_SERVER
+		'tls_server' => STREAM_CRYPTO_METHOD_TLS_SERVER,
+		'tlsv1_1_server' => STREAM_CRYPTO_METHOD_TLSv1_1_SERVER,
+		'tlsv1_2_server' => STREAM_CRYPTO_METHOD_TLSv1_2_SERVER
```

Comment:
These constants are not defined in PHP5.5 and lower. You should use `constant()` to read their values, or conditionally set the keys based on the constants existing.